python/src/aiconfig/editor/client/src/themes/GradioTheme.ts

```diff
@@ -166,6 +167,16 @@ export const GRADIO_THEME: MantineThemeOverride = {
           },
         },
 
+        ".buttonGroupLeft": {
+          borderBottomRightRadius: 0,
+          borderTopRightRadius: 0,
+        },
+
+        ".buttonGroupRight": {
+          borderBottomLeftRadius: 0,
+          borderTopLeftRadius: 0,
+        },
```

Comment:
Why define this separately in Gradio vs. in the component itself?

Reply:
This is just in case we need to explicitly override in gradio if gradio overrides the default